pin/pins/nixpkgs-unstable_0.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I didnt realize there was both a `nixpkgs-unstable` and ` nixos-unstable` branch in `nix-channels` 🤦 Now I know and this looks good!